app/config/roles.php

```diff
@@ -59,6 +61,7 @@
     Auth::USER_ROLE_GUESTS => [
         'label' => 'Guests',
         'scopes' => [
+            'none',
```

Comment:
what is this new scope used for ?

Reply:
No idea. It was giving guest scope missing error if this was not there, at some point. I would be scared to remove it now.

Reply:
Cant remember. Removing it. Lets see if we spot some bug afterwards.